node/sc/bridge_manager.go

```diff
@@ -58,31 +58,20 @@ var (
 
 // RequestValueTransferEvent from Bridge contract
 type RequestValueTransferEvent struct {
-	TokenType    uint8
-	ContractAddr common.Address
-	TokenAddr    common.Address
-	From         common.Address
-	To           common.Address
-	Amount       *big.Int // Amount is UID in ERC721 token
-	RequestNonce uint64
-	URI          string // uri of ERC721 token
-	BlockNumber  uint64
-	Fee          *big.Int
-	txHash       common.Hash
+	*bridgecontract.BridgeRequestValueTransfer
```

Comment:
If `RequestValueTransferEvent` is only pointing the `bridgecontract.BridgeRequestValueTransfer`, how about using the type directly?

Reply:
I tried to use alise type like below.
https://stackoverflow.com/questions/28800672/how-to-add-new-methods-to-an-existing-type-in-go

But, I can't find how to get alias type variable from parent type pointer without hard copy.
I will try to find the way and tell you. :)